src/docs/implementations/dynatrace.adoc

```diff
@@ -66,6 +66,8 @@ MeterRegistry registry = new DynatraceMeterRegistry(dynatraceConfig, Clock.SYSTE
 
 `DynatraceConfig` is an interface with a set of default methods.
 Spring Boot's Micrometer support binds properties prefixed with `management.metrics.export.dynatrace` directly to the `DynatraceConfig`.
+Since Spring Boot version 3.0.0, the prefix used in Spring Boot changed to `management.dynatrace.metrics.export`.
```

Comment:
Does this doc tooling/format offers a way to add a "info" or "warning"? Since this is together with the rest of the text I fear it's not so obvious and folks might just miss it.

Reply:
It does!
There are TIP, NOTE, WARNING and IMPORTANT - see https://micrometer.io/docs/concepts for examples.

Reply:
I am debating whether to use NOTE or IMPORTANT - WDYT?